locales/de/subtitles/erwin-schulz/erwin-schulz-01.yml

```diff
@@ -81,7 +79,7 @@ de:
           s: 15
           from: 00:01:07,06
           to: 00:01:13,05
-          text: The school was not far away from our parents’ flat.
+          text: Die Schule war nicht weit entfernt von unserer elterlichen Wohnung ...
```

Comment:
We try to avoid `...` and the end/beginning of subtitles.
Please just leave the sentence open.

Background: The subtitles are later combined (concatenated) to the Transcript, so we would end up with something like `Die Schule war nicht weit entfernt von unserer elterlichen Wohnung ... ... und Vorbildwirkungen die entwickelten sich später – nicht - mit zunehmendem Alter.`.